Cargo.toml

```diff
@@ -1,10 +1,18 @@
 [package]
 authors = ["Without Boats <woboats@gmail.com>"]
+build = "build.rs"
```

Comment:
I think we don't need this line; we infer that a file called `build.rs` is the build script.